R/zzz.R

```diff
@@ -1,17 +1,18 @@
 .onLoad <- function(...) {
 
   # Only for x64 bits packages. This is required to avoid error when package is being checked on CI for x86
-  is64 <- (.Machine$sizeof.pointer==8)
-  if(!is64){
+  is64 <- (.Machine$sizeof.pointer == 8)
+  if (!is64) {
     return()
   }
-#
+  #
   # Should only be called once
   initPackage()
 
+  ContainerType <- containerType();
 
+  # Ensures that the variable ContainerType is available in the package as ospsuite::ContainerType
+  assign("ContainerType", ContainerType, envir = parent.env(environment()))

```

Comment:
For some reason, doing ContainerType <<- containerType does not allow to retrieve the variable outside of package using ospsuite::ContainerType

Like this, it works

@PavelBal @Yuri05 Not sure if this is the best way to define enum but this works nicely and from a caller, you can have something like

![image](https://user-images.githubusercontent.com/1041237/62007637-f7bc6e00-b11d-11e9-9099-9ad82e988182.png)

Reply:
`ContainerType <<- containerType`

creates `ContainerType` in the `Global Environment`. The way you did it ensures that it is created in the package environment - good!